# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/crashtests/1907464.html
# Commit: 0825e43199d2
# Full Hash: 0825e43199d20888d87c3038413cdb652712a275
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2024-07-16 04:08:30
# Regressor Bug: 1907464
# File Overlap Count: 1
# Description:
#   Bug 1907464 - Clear the shadow-crossing selection when the ParentChainChanged notification is called on the common ancestor r=jjaschke,dom-core
#   
#   It's unclear from the spec about where the new selection should be,
#   so we just clear the selection at the moment.
#   
# ==============================================================================

diff -r 399b1edeca46 -r 0825e43199d2 dom/base/crashtests/1907464.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/base/crashtests/1907464.html	Mon Jul 15 15:29:05 2024 +0000
@@ -0,0 +1,12 @@
+<script>
+window.addEventListener("DOMContentLoaded", () => {
+  let sel = document.getSelection()
+  sel.setPosition(a.offsetParent.attachShadow({mode: "open"}), 0)
+  sel.extend(b, 0)
+  c.replaceWith("1")
+  document.documentElement.innerHTML = document.documentElement.innerHTML
+})
+</script>
+<shadow id="a">
+<style id="b"></style>
+<font id="c">a</font>